src/server/scripts/Commands/cs_modify.cpp

```diff
@@ -525,7 +525,7 @@ class modify_commandscript : public CommandScript
         if (handler->needReportToTarget(target))
             ChatHandler(target->GetSession()).PSendSysMessage(LANG_YOURS_SPEED_CHANGED, handler->GetNameLink().c_str(), Speed);
 
-        target->SetSpeed(MOVE_RUN, Speed, true);
+        target->SetSpeedRate(MOVE_RUN, Speed);
```

Comment:
this is not my code. I just renamed the name of the method and removed one parameter. The main objective of this PR is the changes of implementation of that method.